pkg/fieldpath/fieldpath.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
 )
 
+// formatMap formats map[string]string to a string.
+func formatMap(m map[string]string) string {
+	var l string
+	for key, value := range m {
+		l += key + "=" + value + "\n"
```

Comment:
Actually, just leave it as `key=value` for now. That's not too dissimilar from what kubectl accepts on the command line. We can add json or yaml later, if users request that.